cmake/ReSolveFindHipLibraries.cmake

```
@@ -6,13 +6,20 @@ add_library(resolve_hip INTERFACE)
 find_package(hip REQUIRED)
 find_package(rocblas REQUIRED)
 find_package(rocsparse REQUIRED)
+find_package(rocsolver REQUIRED)
 
-target_link_libraries(resolve_hip INTERFACE
+target_link_libraries(resolve_hip INTERFACE 
   hip::host 
   hip::device
   roc::rocblas
   roc::rocsparse
-  rocsolver
+  roc::rocsolver
 )
```

I had to remove `roc::` namespace in order to build on Frontier. I am not sure what is going on with HIP find functions, but it seems CMake is still a bit buggy.

```suggestion
  rocsolver
)
```